ts-test/tests/put.ts

```diff
@@ -349,18 +349,169 @@ describe('client.put()', function () {
         }).then(() => client.remove(key))
     })
 
-    it('should return a parameter error when bin length exceeds 15 chars', function () {
-      const key: Key = keygen.string(helper.namespace, helper.set, { prefix: 'test/put/' })()
-      const bins: AerospikeBins = { 'bin-name-size-16': 'bin name with 16 chars' }
+    context('should return a parameter error when bin length exceeds 15 chars', function () {
+      it('null value', async function () {
+        const key: Key = keygen.string(helper.namespace, helper.set, { prefix: 'test/put/' })()
+        const bins: AerospikeBins = { 'bin-name-size-16': null }
+
```

Comment:
Does the node.js test framework have a way to parametrize tests? In python you can create a "matrix" of inputs for a test case to avoid repetition